docker-compose.yml

```diff
@@ -239,6 +239,8 @@ services:
         condition: service_completed_successfully
     expose:
       - 3000
+    ports:
+      - 3000:3000
```

Comment:
Do you use the `dory` gem during local development? Or do you go to localhost:3000?

Reply:
`localhost:3000`

Reply:
Accidentally clicked "Unresolve" 🤦